helm/prometheus-agent/charts/prometheus-agent/templates/prometheus.yaml

```diff
@@ -65,6 +65,7 @@ spec:
   {{- else }}
   image: "{{ .Values.global.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
   {{- end }}
+  keepDroppedTargets: {{ .Values.keepDroppedTargets }}
```

Comment:
I don't find `keepDroppedTargets` in PrometheusCR.
https://github.com/search?q=repo%3Aprometheus-community%2Fhelm-charts%20keepDroppedTargets&type=code

Where is it upstream ?

Reply:
https://github.com/prometheus-community/helm-charts/blob/70aa37539496128f1ed07d97e29edbabcc8ded10/charts/kube-prometheus-stack/charts/crds/crds/crd-prometheuses.yaml#L4383